src/main.js

```diff
@@ -55,8 +55,17 @@ if (window.location.search.indexOf("testEnvironment") > -1) {
      * navigator.language in ExtensionLoader (when making require contexts for each
      * extension).
      */
+    var locale;
+    var regex = /locale=([^&]+)&?/.exec(window.location.search);
+    if(window.location.search && regex !== null) {
```

Comment:
Or can you? JS might complain if you do that while declaring an object literal. *tests*